components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/internal/APIManagerComponent.java

```diff
@@ -230,7 +221,23 @@ protected void activate(ComponentContext componentContext) throws Exception {
             APIMgtDBUtil.initialize();
             GatewayArtifactsMgtDBUtil.initialize();
             configureEventPublisherProperties();
-            configureNotificationEventPublisher();
+
+            if (configuration.getEventHubConfigurationDto().isEnabled()) {
```

Comment:
We need to remove this right? Because we need the logging publisher configured if the hub is disabled?

Reply:
Yes.

Reply:
Corrected.